stubs/redis/redis/backoff.pyi

```diff
@@ -1,30 +1,31 @@
 from abc import ABC, abstractmethod
 
 class AbstractBackoff(ABC):
-    def reset(self): ...
+    def reset(self) -> None: ...
     @abstractmethod
-    def compute(self, failures): ...
+    def compute(self, failures: int) -> int: ...
 
 class ConstantBackoff(AbstractBackoff):
-    def __init__(self, backoff) -> None: ...
-    def compute(self, failures): ...
+    def __init__(self, backoff: int) -> None: ...
+    def compute(self, failures: int) -> int: ...
 
 class NoBackoff(ConstantBackoff):
     def __init__(self) -> None: ...
 
 class ExponentialBackoff(AbstractBackoff):
-    def __init__(self, cap, base) -> None: ...
-    def compute(self, failures): ...
+    def __init__(self, cap: int, base: int) -> None: ...
+    def compute(self, failures: int) -> int: ...
 
 class FullJitterBackoff(AbstractBackoff):
-    def __init__(self, cap, base) -> None: ...
-    def compute(self, failures): ...
+    def __init__(self, cap: int, base: int) -> None: ...
+    def compute(self, failures: int) -> int: ...
 
 class EqualJitterBackoff(AbstractBackoff):
-    def __init__(self, cap, base) -> None: ...
-    def compute(self, failures): ...
+    def __init__(self, cap: int, base: int) -> None: ...
+    # Technically it returns `float`,
+    # but `# type: ignore[override]` seems like an overkill.
+    def compute(self, failures: int) -> int: ...
```

Comment:
Wouldn't it make more sense to have `AbstractBackoff.compute()` and this `compute()` return `float` and the other sub-classes return `int`?

Reply:
Or probably everything can be a `float` 🤔 
I will experiment a bit more with this.